numpy/distutils/system_info.py

```diff
@@ -604,30 +604,30 @@ class AliasedOptionError(DistutilsError):
 
 class AtlasNotFoundError(NotFoundError):
     """
-    Atlas (http://github.com/math-atlas/math-atlas) libraries not found.
+    Atlas (https://github.com/math-atlas/math-atlas) libraries not found.
     Directories to search for the libraries can be specified in the
     numpy/distutils/site.cfg file (section [atlas]) or by setting
     the ATLAS environment variable."""
 
 
 class FlameNotFoundError(NotFoundError):
     """
-    FLAME (http://www.cs.utexas.edu/~flame/web/) libraries not found.
+    FLAME (https://shpc.oden.utexas.edu/libFLAME.html) libraries not found.
     Directories to search for the libraries can be specified in the
     numpy/distutils/site.cfg file (section [flame])."""
 
 
 class LapackNotFoundError(NotFoundError):
     """
-    Lapack (http://www.netlib.org/lapack/) libraries not found.
+    Lapack (https://www.netlib.org/lapack/) libraries not found.
     Directories to search for the libraries can be specified in the
     numpy/distutils/site.cfg file (section [lapack]) or by setting
     the LAPACK environment variable."""
 
 
 class LapackSrcNotFoundError(LapackNotFoundError):
     """
-    Lapack (http://www.netlib.org/lapack/) sources not found.
+    Lapack (https://www.netlib.org/lapack/) sources not found.
```

Comment:
```suggestion
    Lapack (https://github.com/Reference-LAPACK/lapack) sources not found.
```

Reply:
Let's not make any changes in numpy.distutils please